processing/src/main/java/io/druid/segment/ObjectColumnSelector.java

```diff
@@ -19,6 +19,8 @@
 
 package io.druid.segment;
 
+import javax.annotation.Nullable;
+
 public interface ObjectColumnSelector<T> extends ColumnValueSelector
```

Comment:
Object in `ObjectColumnSelector` is usually not a Number. It's a number only when numeric columns (long, double, float) are presented as object columns, but there are also "genuinely" object columns (thetaSketch, histogram, etc.)